bats/fb-content-katello.bats

```diff
@@ -121,8 +121,13 @@ EOF
     rpm -e `rpm -qf /etc/rhsm/ca/candlepin-local.pem`
   fi
 
+  yum erase -y 'katello-ca-consumer-*' || true
```

Comment:
These `|| true` have been there since @stbenjam added the code in 0ce8c83a, no idea why :(